tflint-ruleset-blueprint/rules/terraform_required_version.go

```diff
@@ -48,7 +54,30 @@ const (
 
 // Checks if a module has a terraform required_version within valid range.
 func (r *TerraformRequiredVersion) Check(runner tflint.Runner) error {
-	splitVersion := strings.Split(minimumTerraformRequiredVersion, ".")
+	config := &TerraformRequiredVersionConfig{}
+	if err := runner.DecodeRuleConfig(r.Name(), config); err != nil {
+		return err
+	}
+
+	minVersion := minimumTerraformRequiredVersion
+	if config.MinVersion != "" {
+		if _, err := version.NewSemver(config.MinVersion); err != nil {
+			return err
+		} else {
+			minVersion = config.MinVersion
+		}
+	}
+
+	maxVersion := maximumTerraformRequiredVersion
+	if config.MaxVersion != "" {
+		if _, err := version.NewSemver(config.MaxVersion); err != nil {
+			return err
+		} else {
+			maxVersion = config.MaxVersion
+		}
+	}
+
```

Comment:
Given we have options for providing min/max version, could we add a log line here with the final selected versions?

Reply:
Yup, added using the tflint `logger`